modules/lti/README.md

```diff
@@ -0,0 +1,44 @@
+LTI Tools
```

Comment:
Should we add a requirement for the relevant bits to the development environment setup docs?  Specifically, I do not have the following installed:

- selenium
- geckodriver (or chromedriver)
- npm

Reply:
Developers shouldn't usually need to install this.
These tests are not run during the regular build process.
And we are also not listing requirements for other tests.
E.g. you would need `npm` or `yarn` for the admin interface mock or `python`, `mkdocs` and some extensions for building the docs.

It's not that if it is not documented what is required there (and it should be) but it is always documented locally.
Telling developers that they need this when they start Opencast development would just make things more complicated for most people.